packages/@okta/vuepress-site/docs/concepts/multi-tenancy/index.md

```diff
@@ -340,45 +284,43 @@ like the following:
 
 </div>
 
-<div class="padded-line"></div>
-
-> **Note:** okta-dac is not an official Okta product and does not
+> **Note:** okta-dac isn't an official Okta product and doesn't
 qualify for any support. It's recommended to work with your preferred Okta
 Solution Provider to integrate this code within your existing portal.
 
 #### More information
 
-For more information about okta-dac visit
-[here](https://docs.idp.rocks/).
+See [okta-dac](https://docs.idp.rocks/) for more information.
 
 ## Configuration 2
 
-<div class="heading-2-subtext">Host tenants in separate orgs (e.g. Hub and Spoke)</div>
+<div class="heading-2-subtext">Host tenants in separate orgs (for example,
```

Comment:
Looks like you are removing the custom CSS - did you miss one here?

Reply:
Updated in the latest commit.